config/initializers/rack-attack.rb

```diff
@@ -0,0 +1,12 @@
+if OpenProject::Configuration.blacklisted_routes.any?
+  # Block logins from a bad user agent
+  Rack::Attack.blacklist('block forbidden routes') do |req|
+    regex = OpenProject::Configuration.blacklisted_routes.map! { |str| Regexp.new(str) }
+    regex.any? { |i| i =~ req.path }
```

Comment:
i believe the regexp should be changed to `i =~ /\A#{req.path}/`. otherwise you will match any part of the path. e.g. `colors` with also block a path like `/my/favorite-colors/edit` or whatever. was that intended @slawa85 ?